### PR TITLE
feat(logger): disable colors from env

### PR DESCRIPTION
**Pull request by intech**
Related: https://github.com/moleculerjs/moleculer-web/issues/204 and https://github.com/moleculerjs/moleculer-web/issues/138

### CODE REVIEW

**Comment by intech** [comment]
@icebob The error is not related to my changes

File: src/index.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ const http2 					= require("http2");
 const https 					= require("https");
 const queryString 				= require("qs");
 const os 						= require("os");
+
+// Disable global colors logs
+if(String(process.env.COLORS).toLowerCase() === "false") process.env.FORCE_COLOR = 0;
```
Comment (icebob) [inline src/index.js:16]: I don't understand the concept, could you explain why need a `COLORS` env var to set another `FORCE_COLOR` env var?

Comment (icebob) [reply src/index.js:16]: If it's only control the kleur maybe you should send this PR to the [kleur project](https://github.com/lukeed/kleur/blob/master/index.mjs#L10).

Comment (intech) [reply src/index.js:16]: I was wrong. `COLORS` is `MOL_LOGGER__OPTIONS__COLORS` should be for compatibility with env of moleculer-runner.

Problem:
photo_2020-10-27_21-20-40

What is expected:
photo_2020-10-27_21-20-33

**Comment by davidnussio** [comment]
I set FORCE_COLOR=0 in production otherwise kibana logs look messy only for api http requests (registered routes, response status code and response times). The others logs looks good.

moleculer-web use internally kelur but ignore moleculer logger colors configuration.

What do you think about read broker.loggerFactory.opts.options.colors into started method and apply value to kleur.enabled?

**Comment by icebob** [comment]
@davidnussio I've [updated](https://github.com/moleculerjs/moleculer-web/commit/0ee195e37476ee392f6efcd93ba689fe02b197d6#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) the kleur to 4.1.3 and no it consider the FORCE_COLORS flag:
image

**Pull request closed without merging**